src/rubicon/objc/api.py

```diff
@@ -93,11 +94,19 @@
 
 def encoding_from_annotation(f, offset=1):
     argspec = inspect.getfullargspec(inspect.unwrap(f))
+    # typing evaluates a return type of None as <class NoneType>;
+    # convert that back to a None. E721 is the "don't compare types,
+    # use isinstance()" flake8 error. Which is good advice... except
+    # that we're not checking an instance - we *are* comparing types.
```

Comment:
I'm not sure if I entirely understand this comment. Why not deal with `<class NoneType>` as the `type` for None, similar to how `int` is the type of `1`?

We might need to handle this properly when registering the ObjC method later, for example by registering an encoding for `type(None)`, etc:

```
register_preferred_encoding(b"v", type(None))
```

Or is the issue that CFUNCTYPE does deal well with `<class NoneType>` but only `None`?

Reply:
Ill admit I was being a bit lazy here, and just reproducing what the old code was doing. 

You're probably right that we could probably work through the rest of the logic to use NoneType; the only complication I can see is that we can currently rely on the falsiness of None, but we won't be able to if we use NoneType. However, I haven't done enough of a dive into the code to work out if that's a big issue or a minor inconvenience.

Reply:
Lazy works for me.